app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/extrabuildings/ExtraBuildingTypeContainer.kt

```diff
@@ -39,13 +32,15 @@ internal fun ExtraBuildingTypeContainer(
   modifier: Modifier = Modifier,
 ) {
   HedvigCard(modifier) {
-    val nestedScrollConnection = remember {
-      object : NestedScrollConnection {
-        // to intercept scrolling gesture and do not close the sheet while scrolling the list
-        override fun onPostScroll(consumed: Offset, available: Offset, source: NestedScrollSource) = available
-      }
-    }
-    Column(Modifier.nestedScroll(nestedScrollConnection).verticalScroll(rememberScrollState())) {
+//    val nestedScrollConnection = remember {
+//      object : NestedScrollConnection {
+//        // to intercept scrolling gesture and do not close the sheet while scrolling the list
+//        override fun onPostScroll(consumed: Offset, available: Offset, source: NestedScrollSource) = available
+//      }
+//    }
+    Column(
+      // Modifier.nestedScroll(nestedScrollConnection).verticalScroll(rememberScrollState())
+    ) {
```

Comment:
We can just remove this now since it works well right?

Reply:
I was not sure you'd want to use the new element right away:)

Reply:
Yeah especially since this is more of a foundational layout component and not so much a ui design thing, I think we should start using it asap so that we can start making sure that it works well in the first place.